pkg/scheduler/scheduler.go

```diff
@@ -328,6 +328,10 @@ func (sched *Scheduler) preempt(preemptor *v1.Pod, scheduleErr error) (string, e
 				klog.Errorf("Error preempting pod %v/%v: %v", victim.Namespace, victim.Name, err)
 				return "", err
 			}
+			// If the victim is a WaitingPod, send a reject message to the PermitPlugin
+			if waitingPod := fwk.GetWaitingPod(victim.UID); waitingPod != nil {
```

Comment:
We are implementing the preemption interfaces of scheduling framework and I have a question.
Why we delete waiting pods? IMO, we may just need reject them.